web/app/js/components/util/SvgWrappers.jsx

```diff
@@ -37,2590 +37,32 @@ export const grafanaIcon = (
-            L 328.2 298.8
-            Q 326.7 298.1 325.55 297.15 324.1 295.9 323.1 294.25
-            L 323.05 294.25
-            Q 323.2 296.6 323.35 298.5 323.25 298.5 323.2 298.5 320.55 298.75 317.6 298.75 317.6 298.8 317.6 298.85 321.2 298.7 324.55 299.05 324.7 299.05 324.8 299.25 326.3 299.5 328.35 299.2 330.15 299 331.95 298.95 332.25 298.95 332.2 298.75 Z" />
-        <path
-          fill="#040404"
-          fillOpacity="0.984313725490196"
-          stroke="none"
-          d="
-            M 342.45 288.9
-            Q 342.4 288.9 342.35 288.9 342.1 289.45 341.8 289.95 341.6 290.2 341.45 290.45 341.95 290.9 342.8 291 345.3 291.25 347.55 291.7 350.7 292.35 352.9 293.85
-            L 352.9 293.95
-            Q 344.6 296.3 334.65 297.05 334.65 297.1 334.6 297.15
-            L 333.5 298.15
-            Q 333.3 298.25 333.05 298.4 332.75 298.5 332.45 298.6 332.45 298.7 332.45 298.75 342.85 298.25 351.8 296.25 353.35 295.9 354.75 295.35 355.65 293.2 353.9 292.05 351.4 290.4 347.8 289.8 346.55 289.6 345.15 289.25 343.9 289 342.45 288.9 Z" />
-        <path
-          fill="#4B4A4A"
-          fillOpacity="0.996078431372549"
-          stroke="none"
-          d="
-            M 341.45 290.45
-            Q 341.4 290.45 341.4 290.5 340.4 292.55 338.1 293.2 338 293.2 337.95 293.25 337.9 293.25 337.85 293.3 337.5 293.35 337.1 293.4
-            L 337 293.4
-            Q 336.25 295.6 334.65 296.9 334.65 296.95 334.65 297.05 344.6 296.3 352.9 293.95
-            L 352.9 293.85
-            Q 350.7 292.35 347.55 291.7 345.3 291.25 342.8 291 341.95 290.9 341.45 290.45 Z" />
-        <path
-          fill="#E6D4BA"
-          fillOpacity="0.996078431372549"
-          stroke="none"
-          d="
-            M 331.5 270.75
-            Q 331.8 270.5 332.1 270.75 332.9 271.5 333.65 272.35 335.5 274.5 336.7 277.35 337.7 279.75 338.35 282.55 338.9 284.95 338.35 287.65 338.05 289.1 337.85 290.45 337.75 290.75 337.7 291.1 337.6 291.5 337.55 291.9 338.65 291.35 339.5 290.45 340 289.85 340.4 289.15 340.8 288.4 341.05 287.5 341.2 286.95 341.3 286.45 341.9 283.5 341.45 280.25 340.75 275.3 340 271
-            L 340 270.95
-            Q 340.85 272 341.45 273.3 341.45 273.25 341.45 273.2 341.2 272.15 340.9 271.15 339.7 267.3 337.8 264.2 336.35 263.1 335.15 261.75 333.95 260.35 332.4 259.3 327.1 255.7 321.35 252.65 317.8 250.75 313.8 249.3 309.7 247.85 304.95 247.3 297.15 246.35 294.1 251 293.55 251.85 293.5 253 293.45 255.2 293.05 256.9 294.1 260.7 296.35 263.45 297.2 264.5 296.95 265.65 298.9 267.45 301.05 269.05 303.2 270.6 305.6 271.9 308.7 273.6 311.05 276 316.4 281.5 320.1 288.6 321.3 290.9 322.8 292.8 324.35 294.7 326.2 296.25 328.7 298.35 332.45 297.15 334.8 295.1 335.7 291.75 335.9 291.15 335.9 290.75
-            L 335.9 290.7
-            Q 335.9 290.55 335.95 290.45 336.25 285.7 335.3 281.45 334.3 277.05 332.15 273.65 331.25 272.25 331.5 270.75 Z" />
+  <svg width="25px" height="25px" viewBox="-113.76 -96.84 64 64">
```

Comment:
The icon's payload is now sane, but now it looks bigger than Grafana's icon on the screen. From my testing, 17x17 is the sweet spot:

![image](https://user-images.githubusercontent.com/554287/81618318-b3e4dd80-93ac-11ea-91c5-d8f1b3a1175f.png)

Reply:
Fixed it!